scripts/linters/pre_commit_linter.py

```diff
@@ -530,6 +531,10 @@ def main(args=None):
     all_messages += codeowner_linter.check_codeowner_file(
         verbose_mode_enabled)
 
+    all_messages += (
+        third_party_typings_linter.check_third_party_libs_type_defs(
+            verbose_mode_enabled))
```

Comment:
Do not call it here, instead create a new file named `other_linter.py` and add this check under the `OtherLintChecksManager` class.

Reply:
Isn't this a general check like `codeowners` check. That check also is called this way.

Reply:
yep, you are right. Leave this then.